elftools/elf/elffile.py

```diff
@@ -203,12 +227,15 @@ def has_dwarf_info(self):
             self.get_section_by_name('.zdebug_info') or
             self.get_section_by_name('.eh_frame'))
 
-    def get_dwarf_info(self, relocate_dwarf_sections=True):
+    def get_dwarf_info(self, relocate_dwarf_sections=True, follow_links=True):
         """ Return a DWARFInfo object representing the debugging information in
             this file.
 
             If relocate_dwarf_sections is True, relocations for DWARF sections
             are looked up and applied.
+
+            If follow_links is True, we will try to load the supplementary
```

Comment:
Why was the name `follow_links` chosen? Why not something that conveys that this is specifically about supplementary DWARF files?

Reply:
follow_links is used by readelf. I think it is more neutral, as this could be used for other types of external debug files. I haven't looked much into the spec for this, but I think .dwo files could be another case of external files to load.